src/utils/preload.ts

```diff
@@ -14,3 +14,11 @@ export const fixHotReloadNext12 = () => {
     })
   });`);
 };
+
+export function isPlatformMac() {
+    return process.platform === 'darwin';
```

Comment:
curious: Is this method/logic recommended somewhere?

Reply:
https://nodejs.org/api/process.html#process_process_platform , this is nodejs and should provide accurate value ?

is there any problem ?